sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -21,5 +21,7 @@ public class CommunicationTestEnvironment : TestEnvironment
         public string LiveTestConnectionString => GetRecordedVariable(LiveTestConnectionStringEnvironmentVariableName);
 
         public Uri LiveTestEndpoint => new Uri(Core.ConnectionString.Parse(LiveTestConnectionString).GetRequired("endpoint"));
+
+        public string CommunicationTestPhoneNumber => GetOptionalVariable("AZURE_PHONE_NUMBER") ?? "+11234567891";
```

Comment:
`CommunicationTestEnvironment` is for the shared variables. So this should be in the environment type for the phone numbers. Also it should not be optional, or have default. If it is not set, test will fail, right?

Reply:
I found same AZURE_PHONE_NUMBER is in use in SMS, so I made use of it in SMS as well.